.pre-commit-config.yaml

```diff
@@ -73,11 +73,11 @@ repos:
     hooks:
       - id: isort
         name: Run isort for python-sdk
-        args: [ "--profile", "black" ]
+        args: ["--profile=black"]
```

Comment:
Is this necessary?  I like the current way more tbh.

Reply:
reverted